### PR TITLE
chore: enabled model selection in url search tool

### PR DESCRIPTION
**Pull request by gagandt**
## What
Depends on https://github.com/kodu-ai/webapp/pull/52

Added browser models for web search tool, I don't like the naming convention `smart/fast` but I felt it was at least semantically better than giving `claude-3.5-haiku/sonnet`

Always defaults to fast -> sonnet

#### TCs
Screenshot 2024-10-31 at 8 56 37 PM

Screenshot 2024-10-31 at 7 57 41 PM

### CODE REVIEW

File: extension/src/shared/kodu.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import Anthropic from "@anthropic-ai/sdk"
 
-// const KODU_BASE_URL = "http://localhost:3000"
-const KODU_BASE_URL = "https://www.kodu.ai"
+const KODU_BASE_URL = "http://localhost:3000"
```
Comment (ajnart) [inline extension/src/shared/kodu.ts:3]: ^^

**Review by ajnart — CHANGES_REQUESTED** [review]

File: extension/src/shared/kodu.ts (hunk 1 above)
Comment (gagandt) [reply extension/src/shared/kodu.ts:3]: thanks!

Comment (gagandt) [reply extension/src/shared/kodu.ts]: fixed

**Comment by matannahmani** [comment]
i would suggest to move the browser model to settings tab instead of having the AI select it, it makes more sense

**Pull request merged**